scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -353,6 +352,39 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)])
     hotJoined ++ chillJoined
   }
 
+  /**
```

Comment:
Does it require to import any of `algebird` (like for `Hash128`), if so might be worth adding in the doc like in the `skewJoin`.